pom.xml

```diff
@@ -141,17 +141,23 @@
          upgrade) - START -->
       <!-- Newer versions needed than defined in Boot -->
       <!-- Fixed dependencies -->
-      <rabbitmq.http-client.version>3.5.0.RELEASE</rabbitmq.http-client.version>
+      <!-- CVE-2022-25857 -->
+      <snakeyaml.version>1.31</snakeyaml.version>
       <!-- Spring boot version overrides - END -->
 
+      <!-- CVE-2022-31690 -->
+      <spring-security-oauth2-client.version>5.7.6</spring-security-oauth2-client.version>
+
+      <rabbitmq.http-client.version>3.12.1</rabbitmq.http-client.version>
+
       <!-- Vaadin versions - START -->
-      <vaadin.version>8.14.1</vaadin.version>
+      <vaadin.version>8.19.0</vaadin.version>
```

Comment:
8.19 and onwards is under a commercial license ([Vaadin Commercial License version 1](https://vaadin.com/commercial-license-and-service-terms)): 
https://github.com/vaadin/framework/blob/master/CHANGELOG-VAADIN8.md

8.18 is the latest we can use without a subscription if I understood the terms correctly.

Reply:
yes, that's really a pity. I will turn it back to 8.14.3.